src/debug.c

```diff
@@ -1087,6 +1089,9 @@ NULL
             addReplySubcommandSyntaxError(c);
             return;
         }
+    } else if (!strcasecmp(c->argv[1]->ptr, "set") && c->argc == 4) {
+        setKey(c,c->db, c->argv[2], c->argv[3], SETKEY_ADD_OR_UPDATE);
```

Comment:
to be honest, even here we doing test, but i don't feel it is a good idea to set the argv to the db.  in https://github.com/redis/redis/pull/13806, we also want to distinguish the normal object and key value object 
BTW, here we also need to `decrRefCount` for c->argv[3]

Reply:
probably a better approach might be having a command like `debug bypass-slot-check yes/no`, set a debug variable and skip slot check if it is enabled. This way, you can go through regular command path and run all kind of commands, not just set.

Reply:
makes sense

Reply:
@ShooterIT I do not understand your comment, sorry. Can you please elaborate a little? The whole idea here is to set a key on a slot that is not owned by this node (for testing purposes).